SAMtags.tex

```diff
@@ -59,13 +60,16 @@ \section{Standard tags}
   {\tt BC} & Z & Barcode sequence identifying the sample \\
   {\tt BQ} & Z & Offset to base alignment quality (BAQ) \\
   {\tt BZ} & Z & Phred quality of the unique molecular barcode bases in the {\tt OX} tag \\
+  {\tt CB} & Z & Sequence bases of the (possibly corrected) cellular-barcode sequence\\
```

Comment:
Looking at some recent cellranger data, I see CR and CB tags like "GCGCAGTTCTCGAGTA" and "GCGCAGTTCTCGAGTA-1".  Ie CB isn't just an error corrected version, but is a UMI-like identifier that contains more than just the sequence.

Note this isn't how 10x describe it on their web pages either: https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/output/bam

Reply:
I think that the CB tag description in the PR is more general and subsumes the 10x description (provided in the above link). Furthermore it is consistent with other tags that may be corrected.